packages/grant-explorer/src/features/projects/ViewProject.tsx

```diff
@@ -0,0 +1,303 @@
+                  />
+                  <div className="[&_a]:underline">
+                    {projectDetailsTabs[selectedTab].content}
+                  </div>
+                </>
+              ) : (
+                <p>Couldn't load project data.</p>
+              )}
+            </div>
+          </div>
+        </DefaultLayout>
+      ) : (
+        <NotFoundPage />
+      )}
+    </>
+  );
+}
+
+function ProjectDetailsTabs(props: {
+  tabs: string[];
+  onChange?: (tabIndex: number) => void;
+  selected: number;
+}) {
+  return (
+    <Box className="" bottom={0.5}>
+      {props.tabs.length > 0 && (
+        <Tabs
+          display="flex"
+          onChange={props.onChange}
+          defaultIndex={props.selected}
+        >
+          {props.tabs.map((tab, index) => (
+            <Tab key={index}>{tab}</Tab>
+          ))}
+        </Tabs>
+      )}
+    </Box>
+  );
+}
+
+function ProjectLinks({ project }: { project?: v2Project }) {
+  const {
+    metadata: {
+      createdAt,
+      updatedAt,
+      website,
+      projectTwitter,
+      projectGithub,
+      userGithub,
+    },
+  } = project ?? { metadata: {} };
+
+  const createdOn =
+    createdAt &&
+    `Created on: ${formatDateWithOrdinal(new Date(createdAt ?? 0))}`;
+
+  const updatedOn =
+    updatedAt &&
+    `Last edited: ${formatDateWithOrdinal(new Date(updatedAt ?? 0))}`;
+
+  return (
+    <div
+      className={`grid md:grid-cols-2 gap-4  border-y-[2px] py-4 my-4 ${
+        // isLoading?
+        createdAt ? "" : "bg-grey-100 animate-pulse"
+      }`}
+    >
+      <ProjectLink url={website} icon={GlobeIcon}>
+        {website}
+      </ProjectLink>
+      <ProjectLink icon={CalendarIcon}>{createdOn}</ProjectLink>
+      {projectTwitter !== undefined && (
+        <ProjectLink
+          url={`https://twitter.com/${projectTwitter}`}
+          icon={TwitterIcon}
+          isVerified={false}
+        >
+          {projectTwitter}
+        </ProjectLink>
+      )}
+      <ProjectLink icon={CalendarIcon}>{updatedOn}</ProjectLink>
+      {projectGithub !== undefined && (
+        <ProjectLink
+          url={`https://github.com/${projectGithub}`}
+          icon={GithubIcon}
+          isVerified={false}
+        >
+          {projectGithub}
+        </ProjectLink>
+      )}
+      {userGithub !== undefined && (
+        <ProjectLink url={`https://github.com/${userGithub}`} icon={GithubIcon}>
+          {userGithub}
+        </ProjectLink>
+      )}
+    </div>
+  );
+}
+
+function ProjectLink({
+  icon,
+  children,
+  url,
+  isVerified,
+}: PropsWithChildren<{
+  icon: FunctionComponent<ComponentPropsWithRef<"svg">>;
+  url?: string;
+  isVerified?: boolean;
+}>) {
+  const Component = url ? "a" : "div";
+  return children ? (
+    <div className="flex items-center gap-2">
+      <div>{createElement(icon, { className: "w-4 h-4 text-grey-400" })}</div>
+      <div className="flex gap-2">
+        <Component
+          href={url}
```

Comment:
@bhargavaparoksham  `href` is not a valid attribute for a `div` element. it's better create the 2 possible elements independently